.github/workflows/build.yml

```diff
@@ -50,11 +54,18 @@ jobs:
           key: ${{ matrix.target }}-${{ steps.get_time.outputs.timestamp }}
           restore-keys: ${{ matrix.target }}-
 
-      - name: Install dependencies
+      # For LuaJIT build
+      - name: Enable multilib
+        if: ${{ matrix.target == 'i686-w64-mingw32' }}
         run: |
           sudo dpkg --add-architecture i386
           sudo apt-get update
-          sudo apt-get install -y ccache g++-mingw-w64 gcc-multilib nasm ninja-build pkg-config python3-pip wine wine32 wine64
+          sudo apt-get install -y gcc-multilib
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ccache g++-mingw-w64 nasm ninja-build pkg-config wine wine64
```

Comment:
opportunity to make wine install conditional only if needed

Reply:
Done in separate commit, because it is easier after the switch to 24.04.